newIDE/app/src/AssetStore/ExtensionStore/ExtensionsSearchDialog.js

```diff
@@ -101,18 +101,25 @@ export default function ExtensionsSearchDialog({
             isInstalling={isInstalling}
             onInstall={async extensionShortHeader => {
```

Comment:
should we move this function out of the render to have more clarity on what it does?

Reply:
Good idea, I'll move it in my next PR (the yml is not correctly parsed)